Tools/Scripts/generate-static-analysis-archive

```diff
@@ -82,28 +87,41 @@ def get_total_issue_count(project_dirs, static_analysis_dir):
     return total_issue_count
 
 
+def create_results_file(options, output_root, static_analysis_dir, new=False):
+    project_dirs = list(filter(lambda x: x[0] != '.' and os.path.isdir(os.path.join(static_analysis_dir, x)),
+                                         os.listdir(static_analysis_dir)))
+
+    if options.id_string:
+        results_page = generate_results_page(project_dirs, options.id_string, output_root, static_analysis_dir, new)
+        if new:
+            f = open(output_root + '/new-results.html', 'w')
+        else:
+            f = open(output_root + '/results.html', 'w')
+        f.write(results_page)
+        f.close()
+
+
 def main(options):
     output_root = options.output_root
     static_analysis_dir = output_root
 
     if os.path.isdir(os.path.join(static_analysis_dir, 'StaticAnalyzer')):
         static_analysis_dir = os.path.join(static_analysis_dir, 'StaticAnalyzer')
+    
+    create_results_file(options, output_root, static_analysis_dir)
 
-    project_dirs = list(filter(lambda x: x[0] != '.' and os.path.isdir(os.path.join(static_analysis_dir, x)),
-                                         os.listdir(static_analysis_dir)))
-
-    if options.id_string:
-        results_page = generate_results_page(project_dirs, options.id_string, output_root, static_analysis_dir)
-        f = open(output_root + '/results.html', 'w')
-        f.write(results_page)
-        f.close()
+    if os.path.isdir(os.path.join(output_root, 'StaticAnalyzerRegressions')):
+        static_analysis_new_dir = os.path.join(output_root, 'StaticAnalyzerRegressions')
+        create_results_file(options, output_root, static_analysis_new_dir, new=True)
```

Comment:
Hm... `new=True` doesn't really convey the semantics here. `is_new_result=True` maybe?